cinch/playbooks/jenkins_backup.yml

```diff
@@ -0,0 +1,97 @@
+# This playbook is specifically provided for scenarios where cinch will be used
+# to manage configurations of Jenkins masters that were *not* created by cinch,
+# but by some other method, either manually or via another configuration
+# management tool.  In this case, it's highly recommended to configure backups
+# before running cinch against a Jenkins master for the first time.
+
+# For this playbook to succeed, the thinBackup plugin must be installed, a
+# valid Jenkins user with admin rights must be provided, along with a target
+# directory for the backup data to live.
+
+# It's important to avoid submitting a Jenkins admin username that does not
+# exist on the target Jenkins master.  If this happens, then the
+# jenkins_user_api module will happily create a useless Jenkins user with the
+# name given, which is best avoided.
+
+# Example:
+# ansible-playbook -u root --private-key /path/to/key
+# -i 'jenkins.example.com', jenkins_backup.yml
+# -e "jenkins_admin=example_username"
+# -e "backup_dir=/var/lib/jenkins/thinBackup_data"
+
+- hosts: all
+  vars:
+    jenkins_url: "http://localhost:8080"
+    jenkins_user: jenkins
+    jenkins_backup:
+      directory: "{{ backup_dir }}"
+      full_schedule: H 0 * * 0
+      diff_schedule: H 0 * * *
+      max_sets: 2
```

Comment:
With regards to all the thinBackup bits of this PR, we have disabled the execution of this code, but are merging as-is for now with the intention to re-visit the backup strategy very soon.  Please understand that the feedback here is very valuable and that we will ensure that everyone is happy with the configuration before re-enabling the thinBackup code, if we decide to do so.